pkg/kubelet/cm/deviceplugin/endpoint_test.go

```diff
@@ -54,22 +54,60 @@ func TestRun(t *testing.T) {
 		{ID: "AThirdDeviceId", Health: pluginapi.Healthy},
 	}
 
-	p, e := esetup(t, devs, socket, "mock", func(n string, a, u, r []pluginapi.Device) {
-		require.Len(t, a, 1)
-		require.Len(t, u, 1)
-		require.Len(t, r, 1)
-
-		require.Equal(t, a[0].ID, updated[1].ID)
-
-		require.Equal(t, u[0].ID, updated[0].ID)
-		require.Equal(t, u[0].Health, updated[0].Health)
+	callbackCount := 0
```

Comment:
Those will get changed by https://github.com/kubernetes/kubernetes/pull/52616
Forgot about that sorry :)

Reply:
All right, i will rebase your code :)